### PR TITLE
Add CHANGELOG for v0.4

### PR DESCRIPTION
**Pull request by misraved**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## v0.4 [2021-05-28]
+
+_Bug fixes_
+
+- Duplicate local variables have now been removed across all the files
+- `README` file now includes the correct github url for cloning the repository
```
Comment (rajlearner17) [inline CHANGELOG.md:6]: `README` file now includes the https github url for cloning the repository

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## v0.4 [2021-05-28]
+
+_Bug fixes_
+
+- Duplicate local variables have now been removed across all the files
```
Comment (rajlearner17) [inline CHANGELOG.md:5]: Duplicate or Unused?

Comment (misraved) [reply CHANGELOG.md:5]: Duplicate, since the local variable has been repeated across the files

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**